usr/lib/cca_stdll/cca_specific.c

```diff
@@ -3548,6 +3548,234 @@ CK_RV token_specific_aes_xts_key_gen(STDLL_TokData_t *tokdata, TEMPLATE *tmpl,
     return CKR_OK;
 }
 
+static CK_RV import_aes_xts_key(STDLL_TokData_t *tokdata,
+                                OBJECT * object)
+{
+    CK_RV rc;
+    CK_ATTRIBUTE *opaque_attr = NULL;
+    enum cca_token_type token_type, token_type2;
+    unsigned int token_keybitsize, token_keybitsize2;
+    const CK_BYTE *mkvp, *mkvp2;
+    CK_BBOOL new_mk;
+
```

Comment:
I guess you should also check `cca_private->pkey_wrap_supported` here. 
We should not allow to import XTS keys if we have no CCA card that supports it.